ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -18,7 +18,7 @@ trait JdbcConnectionProvider extends AutoCloseable {
     * is returned to the pool.
     * The block must not recursively call [[runSQL]], as this could result in a deadlock
     * waiting for a free connection from the same pool. */
-  def runSQL[T](block: Connection => T): T
+  def runSQL[T](description: String, block: Connection => T): T
```

Comment:
Perhaps `runSQL[T](description: String)(block: Connection => T): T`?

This way it can be called like this:

```scala
runSQL("some description") { connection =>
  ...
}
```

Reply:
Interestingly that's the version that I had in our repo, but then I noticed that [SQLExecutor](https://github.com/digital-asset/daml/blob/master/ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/util/SqlExecutor.scala#L36) combined the parameters in one list, so I combined them here as well.

Reply:
Removed the HikariJdbcConnectionProvider.description parameter as part of moving the error logging from HikariJdbcConnectionProvider to SqlExecutor